README.md

```diff
@@ -5,19 +5,69 @@ The mission planner is based on the popular state machine library `smach`, but n
 
 Some terminology to start with:
 - In the context of this package, a waypoint denotes a named pose in the global frame.
-- A mission is a collection of tasks which are executed sequentially (e.g. a path through several waypoints).
-- A mission plan defines the logic how and when different missions are executed. The mission plan is implemented as a `smach` state machine.
-- Each mission has its own mission data, containing relevant information for the mission (e.g. its waypoints).
+- A mission is a collection of tasks that are executed sequentially (e.g. a path through several waypoints).
+- A mission plan defines the logic of how and when different missions are executed. The mission plan is implemented as a `smach` state machine.
+- Each mission has its mission data, containing relevant information for the mission (e.g. its waypoints).
 
 ## Installation
 - Clone this repo into your workspace. _If you followed the instructions on how to install the SMB software, the source code of this package is already located in your workspace._
-- Install `smach_ros` from [here](http://wiki.ros.org/smach_ros) (a state machine library written in python with ROS support)
+- Install `smach_ros` from [here](http://wiki.ros.org/smach_ros) (a state machine library written in Python with ROS support)
 - Install `yaml` with `pip install pyyaml` 
 - (Buid the package with `catkin build smb_mission_planner`)
 
+## Executing your mission
 
+### Basic features
+The `mission_planner.py` executes the previously defined mission plan.
+
+Start the simulation and the path planner.
+As soon as the path planner is ready to receive goals, start the `mission_planner` with
+```
+roslaunch smb_mission_planner mission_planner.launch
+```
+The robot will now try to reach the specified waypoints of each mission one by one, as defined in the `yaml` file.
+In the command line window of the `mission_planner` you can find information on where and in which mission you currently are.
+
+### Examples
+In the folder `src/smb_mission_planner/missions` you can find two mission examples:
+- `twist_mission`: this mission executes a set of twist commands defined in the mission's config file.
+- `waypoint_mission`: this mission publishes waypoints defined in the mission's config file to the path planner
+
+To add or remove this mission, please modify the file `src/smb_mission_planner/mission_planner.py`. This can be done by adding or removing the mission's class to the state machine in the function `createStateMachine(self)`.
+
+```python
+# Add waypoint mission
+smach.StateMachine.add('Waypoint Mission', WaypointMission(self.missions_data['waypoint_mission'], self.reference_frame), transitions={'Completed': 'Twist Mission', 'Aborted': 'Failure', 'Next Waypoint': 'Waypoint Mission'})
```

Comment:
somehow, this transitions still confuse me.
After completion of the Waypoint Mission, the Twist Mission is executed?

Reply:
This is only an example of how you can run multiple type of missions one after the other. I you want to finish you can substitute `Completed : Twist Mission` with `Completed: Success`.

Reply:
Wouldn't that be worth a comment?